sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/AggregateScanConverter.java

```diff
@@ -234,4 +237,41 @@ private AggregateCall convertAggCall(
     return AggregateCall.create(
         sqlAggFunction, false, false, false, argList, -1, RelCollations.EMPTY, returnType, aggName);
   }
+
+  private SqlAggFunction getBuiltinFunction(ResolvedAggregateFunctionCall aggregateFunctionCall) {
```

Comment:
nit: In my opinion moving these into functions reduces the readability of this code. These are small functions called in exactly one place, consider inlining and adding a comment if you think it is needed.

Reply:
done